utils/frame/rpc-utils/src/lib.rs

```diff
@@ -0,0 +1,245 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Shared JSON-RPC client related code and abstractions.
+
+use async_trait::async_trait;
+use serde::de::DeserializeOwned;
+use sp_runtime::traits::{Block as BlockT, Header as HeaderT};
+use std::collections::VecDeque;
+
+pub use jsonrpsee::{
```

Comment:
is there actually any logic change here as well?

Reply:
I have increased  `connection timeout` and `request timeout` but nothing else.

In addition I removed this `keep connection` stuff completely instead if one wants make a request then a client needs instantiated otherwise it's not possible to get access the RPC trait stuff from the macros.